packages/openactive-integration-tests/documentation/generator.js

```diff
@@ -46,26 +55,29 @@ global.afterEach = () => {};
 global.documentationGenerationMode = true;
 
 // Load metadata from all tests
-var tests = fg.sync(pkg.jest.testMatch, { cwd: rootDirectory }).map(function(file) {
-  console.log('Reading: ' + file);
+const tests = fg.sync(pkg.jest.testMatch, { cwd: rootDirectory }).map(function (file) {
+  console.log(`Reading: ${file}`);
   // TODO: Verify that the data actually conforms to the type.
-  return /** @type {TestModuleExports} */(require(`${rootDirectory}${file}`));
+  const data = /** @type {TestModuleExports} */(require(`${rootDirectory}${file}`));
+  const expectedPath = `test/features/${renderFullTestPath(data)}`;
+  chai.expect(expectedPath).to.equal(file, `Expected ${file} to contain metadata matching its path`);
```

Comment:
Added a little something to check the feature metadata identifiers match the test file path when the docs are generated (and fixed some lint issues with the file at the same time)